docs/gallery_code/general/plot_anomaly_log_colouring.py

```diff
@@ -13,17 +13,17 @@
 
 To do this, we create a custom value mapping function (normalization) using
 the matplotlib Norm class `matplotlib.colours.SymLogNorm
-<https://matplotlib.org/api/_as_gen/matplotlib.colors.SymLogNorm.html#matplotlib.colors.SymLogNorm>`_.
+<https://matplotlib.org/stable/api/_as_gen/matplotlib.colors.SymLogNorm.html>`_.
```

Comment:
Is there a reason this example can't use intersphinx as well?

Reply:
@QuLogic Fair point, I'll try it. 

That would certainly be a more robust approach overall 👍

Reply:
intersphinx doesn't seem to support `#anchor`, which is surprising... that or the syntax isn't obvious.

But I've changed what I could.